scripts/cloudformation/bitops.schema.yaml

```diff
@@ -18,6 +18,12 @@ cloudformation:
         cfn-stack-name:
           type: string
           export_env: CFN_STACK_NAME
+        s3bucket:
+          type: string
+          export_env: CFN_TEMPLATE_S3_BUCKET
+        s3prefix:
+          type: string
+          export_env: CFN_S3_PREFIX
```

Comment:
1. Add these to https://github.com/bitovi/bitops/blob/add-cf-nested-templates-processing/docs/tool-configuration/configuration-cloudformation.md
2. (Optional) is there a better name than `s3bucket` and `s3prefix` that explains what these keys are for?